frontend/src/v5/ui/routes/viewer/tickets/ticketDetails/ticketsDetailsCard.component.tsx

```diff
@@ -62,6 +58,14 @@ export const TicketDetailsCard = () => {
 	const goPrev = () => changeTicketIndex(-1);
 	const goNext = () => changeTicketIndex(1);
 
+	const goBack = () => {
+		// TODO - replace `getTicketIsCompleted` with <not part of the filtered tickets list>
+		if (getTicketIsCompleted(ticket)) {
```

Comment:
Oh boy this is going to cause conflicts with my pin filtering PR lol. I'll need to make sure my PR does this with the new filtering logic.

One problem with this line in particular is that if you are viewing a completed ticket and exit to the tickets list it will increase the selected index by 1 which means a seemingly random ticket is selected when no values were even changed.

I think this should be easy to fix in my PR (I think this is what you mean with your TODO) because the filtering persists whatever state the tickets card is in. This allows me to use only the filtered tickets when cycling between tickets. So you can just use whether the ticket _id still exists in the filtered tickets to determine whether the index should change.

I'm happy to ignore the problem with getTicketIsCompleted since it will get changed in my PR anyway